apps/admin-ui/src/identity-providers/add/DescriptorSettings.tsx

```diff
@@ -374,6 +376,41 @@ const Fields = ({ readOnly }: DescriptorSettingsProps) => {
               )}
             ></Controller>
           </FormGroup>
+          <FormGroup
+            label={t("encryptionAlgorithm")}
+            labelIcon={
+              <HelpItem
+                helpText={th("encryptionAlgorithm")}
+                fieldLabelId="identity-provider:encryptionAlgorithm"
+              />
+            }
+            fieldId="kc-encryptionAlgorithm"
+          >
+            <Controller
+              name="config.encryptionAlgorithm"
+              defaultValue="RSA-OAEP"
+              control={control}
+              render={({ field }) => (
+                <Select
+                  toggleId={"kc-encryptionAlgorithm"}
```

Comment:
This can be a literal string. I guess we should set up a linting rule for this at some point.
```suggestion
                  toggleId="kc-encryptionAlgorithm"
```